src/main/java/seedu/address/model/quiz/Quiz.java

```diff
@@ -172,6 +171,40 @@ public boolean toggleIsCardDifficult(int index) {
         return sessionCard.isCardDifficult();
     }
 
+    /**
+     * Gets the question header if not flipped, else gets answer header
+     */
+    public String getQuestionHeader() {
+        QuizCard original = currentSession.get(currentQuizCard.getIndex());
+
+        if (original.getQuestion().equals(currentQuizCard.getQuestion())) {
+            return original.getQuestionHeader();
+        }
+
+        return original.getAnswerHeader();
+    }
+
+    /**
+     * Gets the answer header if not flipped, else gets question header
+     */
+    public String getAnswerHeader() {
+        QuizCard original = currentSession.get(currentQuizCard.getIndex());
+
+        if (original.getAnswer().equals(currentQuizCard.getAnswer())) {
+            return original.getAnswerHeader();
+        }
+
+        return original.getQuestionHeader();
+    }
+
+    public List<String> getOpt() {
+        assert currentCardIndex != -1;
+
+        QuizCard original = currentSession.get(currentQuizCard.getIndex());
+
+        return original.getOpt();
```

Comment:
Should inline variable.

```suggestion
        return currentSession.get(currentQuizCard.getIndex()).getOpt();
```

Reply:
Following Law of Demeter, hence I had a local variable instead of using inline variable